scripts/repository.lic

```diff
@@ -1352,7 +1373,7 @@ elsif (cmd[0] =~ /^info$/i)
       output.concat "available versions: #{sorted_versions}\n\n"
     end
     output.concat "#{data}\n\n"
-    respond output
+    puts output
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`

Reply:
Looks like you missed one still.